Tech Lead/Minutes/20240213.md

```diff
@@ -0,0 +1,43 @@
+## Technical Lead Call 13 February 2024 (online; one weeks after design date)
+
+## Participants
+- Attendees: @headmetal @rbeucher @flicj191 @ehogan @valeriupredoi @bouweandela
+- Absentees: @sloosvel @schulnam (DLR forced holiday)
+- Chairs and minutes takers: @valeriupredoi
+- Special mention: first meeting for @headmetal @rbeucher @flicj191 (welcome, folks!)
+
+  - Names legend:
+    - Bouwe Andela [@bouweandela](https://github.com/bouweandela)
+    - Saskia Sloosveldt [@sloosvel](https://github.com/sloosvel)
+    - Valeriu Predoi [@valeriupredoi](https://github.com/valeriupredoi)
+    - Manuel Schlund [@schlunma](https://github.com/schlunma)
+    - Emma Hogan [@ehogan](https://github.com/ehogan)
+    - Felicity Chun [@flicj191](https://github.com/flicj191)
+    - Mike Tetley [@headmetal](https://github.com/headmetal)
+    - Romain Beucher [@rbeucher](https://github.com/rbeucher)
+
+## Proposed topics for discussion (to be ordered then added to the Agenda section at the start of the meeting)
+
+- Review of Pending Action Items
+- Update about CMEW @ehogan
+- OBS data versioning [ESMValGroup/ESMValTool#3452](https://github.com/ESMValGroup/ESMValTool/issues/3452) @bouweandela
+- Managing connections and dealing with knowledge gaps when personnel move @bouweandela
+
+## Topics discussed
+
+- @valeriupredoi: TLT meeting scheduling
+- Next release: likely in April/May by @ehogan and team
+  - @rbeucher is asking what we do for a release: @bouweandela giving a brief overview of the workload needed
+  - TLT members have left/moved, so we are short on people
+  - @ehogan - MetO still working on the RTW (recipe test workflow) - it works on a small scale (prototype)
+  - recipes break since dependencies change but the maintainers don't run them periodically
+  - @rbeucher - could run different recipes on different machines to speed the process up: comparison vs previous run could still be made?
+- OBS data versioning [ESMValGroup/ESMValTool#3452](https://github.com/ESMValGroup/ESMValTool/issues/3452) @bouweandela - to be looked at by @valeriupredoi as well
+- CMEW update (V will write a short paragraph offline)
```

Comment:
TODO?

Reply:
Yes just like the shared email 🤣 Cheers, bud, will do in the morrow